### PR TITLE
Next going live

### PR DESCRIPTION
**Pull request by EmilK15**

### CODE REVIEW

**Comment by doitadrian** [comment]
Is there a Netlify link for this?

Weird, all other PRs had it. 🤔

**Comment by doitadrian** [comment]
Maybe I'll manage to review it via https://github.com/webiny/docs.webiny.com/blob/310dce44bf68b025db082b809609e48509f19c1d/docs/get-started/going-live.md :D

**Comment by EmilK15** [comment]
> Is there a Netlify link for this? @EmilK15
> 
> Weird, all other PRs had it. 🤔

This was before Netlify was setup for `next` branch

**Comment by EmilK15** [comment]
> The changes proposed here look good to me.
> 
> However, I really think a few minor things can be improved. Warning: some of these might be subjective :)
> 
> 1. Please check:
> 
> > This article demonstrates how you can deploy your project to dev, prod and any other environment you may have. At the end of this process, you will have a public URL to show your website to the world.
> 
> Grammarly tells me it's `At the end`, not `In the end`.
> 
> 1. A few comments on this:
> 
> image
> 
> A. I would prefix this sentence with `After that, `. It really sounds awkward when I read current text to myself. I think it can be written more pleasantly. :)
> 
> B. For a moment, I was like "wait, which `env.json` is this?". So, either we prefix this `.env.json` with `/`, `{projectRoot}`, or maybe we just append "located in the project root". Just so the user immediately knows which `env.json` file we are referring to here.
> 
> 1. I would just prepend this sentence with "And also, ". Without it, when reading the text, it feels a bit robotic :D
> 
> image

See new changes. Also removed the AWS credentials that we don't need anymore

File: docs/get-started/going-live.md [hunk 1]
```diff
@@ -49,22 +53,23 @@ However, if you want to use a different database, for example, you can create a
...
-webiny deploy-apps --env=dev
+webiny deploy apps --env=dev
 ```
 
 ## Other environments
 
 If you also want to deploy a `prod` environment, repeat the procedure using the `prod` value.
 
-If you need more environments, you can easily add those following the same procedure, but you will need to also add the appropriate build scripts to your apps, in the following files:
+This can be extended if you need more environments, you can easily add them in the same procedure shown above, but you will need to also add the appropriate build scripts to your apps, in the following files:
```
Comment (doitadrian) [inline docs/get-started/going-live.md:72]: Please put the production environment deployment into its own section. For example `Deploying to prod`.

Then, let's use this content for the `Other environments` section:

## Other environments
Apart from seen `dev` and `production` environments, Webiny also supports adding new ones. In order to do so, the same steps described above are to be taken, with one additional step and that is inserting a build script for existing React apps, in their respective `package.json` files:

```
apps/admin/package.json
apps/site/package.json
apps/another-app/package.json
```

So, an example script for a `staging` environment could look like the following:

```
"build:staging": "env-cmd -r .env.json -e default,staging webiny run build"
```

Once this is inserted, as usual, the following command should successfully execute a `staging` environment deployment:

```
webiny deploy apps --env=staging
```

File: docs/get-started/going-live.md [hunk 2]
```diff
@@ -15,31 +15,35 @@ Open the `api/.env.json` file. You should see a similar config file:
...
-    "MONGODB_NAME": "webiny-72858e17",
-    "MONGODB_SERVER": "mongodb+srv://username:password@my-db-123.mongodb.net/test?retryWrites=true",
     "S3_BUCKET": "webiny-files-72858e17",
     "JWT_SECRET": "GSyWyuHU2uHrO+wnSO8OjQ+wPbuQmBclVNFejoo9qmsQRNFAZHu+PFSyq48+"
   }
 }
 ```
 
-If you don't want to change anything for your `dev` environment, leave it as is.
+In the top-level of the project, look at the `.env.json` file. You should something similar:
```
[the remark this answers is not in the document]
Comment (EmilK15) [reply docs/get-started/going-live.md]: 👍

File: docs/get-started/going-live.md (hunk 1 above)
Comment (EmilK15) [reply docs/get-started/going-live.md]: Added

File: docs/get-started/going-live.md [hunk 3]
```diff
@@ -49,22 +53,23 @@ However, if you want to use a different database, for example, you can create a
...
 From the root of your project, run the following command:
 
 ```
-webiny deploy-apps --env=dev
+webiny deploy apps --env=dev
 ```
 
 ## Other environments
 
 If you also want to deploy a `prod` environment, repeat the procedure using the `prod` value.
```
[the remark this answers is not in the document]
Comment (EmilK15) [reply docs/get-started/going-live.md]: added to the new commit

**Pull request merged**